converters/common.js

```diff
@@ -119,6 +119,31 @@ const TuyaDataTypes = {
     bitmap: 5, // [ 1,2,4 bytes ] as bits
 };
 
+const silvercrestModes = {
+    white: 0,
+    color: 1,
+    scene: 2,
+};
+
+const silvercrestEffects = {
```

Comment:
Should this not be consistent with the naming above? Both scene to match the app, or effect to match closer to zigbee?

Reply:
On a personal note: scene might be confusing with actually scene support in zigbee. But I don’t have strong feeling either way, aside from that it should be consistent.

Reply:
> 
> 
> Should this not be consistent with the naming above? Both scene to match the app, or effect to match closer to zigbee?

I agree. Changed it.

Reply:
> On a personal note: scene might be confusing with actually scene support in zigbee. But I don’t have strong feeling either way, aside from that it should be consistent.

This is a good point, scene conflicts with https://www.zigbee2mqtt.io/information/scenes.html , in Zigbee2MQTT such things are called `effect` (https://www.zigbee2mqtt.io/information/mqtt_topics_and_message_structure.html). I think it will be good to change to `effect`.

Reply:
Changed it. Should work but I'll test it functionaly in a few hours.

Reply:
Yea still works. I also renamed it in the PR for the docs.